cypress/support/fragments/invoices/invoices.js

```diff
@@ -70,20 +73,17 @@ export default {
 
   deleteInvoiceViaActions() {
     cy.do([
-      PaneHeader({ id: 'paneHeaderpane-invoiceDetails' })
+      PaneHeader({ id: invoiceDetailsPaneId })
         .find(actionsButton).click(),
       Button('Delete').click(),
       Button('Delete', { id:'clickable-delete-invoice-confirmation-confirm' }).click()
```

Comment:
wrap to special method with confirmation

Reply:
Could you please clarify what exactly needs to be wrapped? I didn't get it.

Reply:
Updated